Kommunicate-io/Kommunicate-iOS-SDK
# [CM-1081] Added pseudonym support

## Summary
- Added pseudonym support in IOS SDK.

<img width="313" alt="SCR-20230914-qkln" src="https://github.com/Kommunicate-io/Kommunicate-iOS-SDK/assets/139108613/e2b3c154-2df5-4969-9bdd-272ee299a869">

## Testing 
- Enable pseudonym support from [here.](https://dashboard.kommunicate.io/settings/pseudonym)
- Login as visitor and create a new chat
- Check in the dashboard whether a pseudonym is visible instead of a random id or not.
